certified-connectors/UiPath Orchestrator/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3382 @@
+{
+  "swagger": "2.0",
+  "info":
+  {
+    "title": "UiPath Orchestrator",
+    "description": "Run automations in UiPath Cloud Orchestrator.",
+    "version": "1.0.0",
+    "contact":
+    {
+      "name": "Integrations Team",
+      "email": "integrations-apps@uipath.com",
+      "url": "https://www.uipath.com/support"
+    }
+  },
+  "host": "cloud.uipath.com",
+  "basePath": "/orchestrator_/",
+  "schemes":
+  [
+    "https"
+  ],
+  "x-ms-connector-metadata":
+  [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.uipath.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.uipath.com/legal/privacy-policy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "AI;Business Intelligence"
+    }
+  ],
+  "securityDefinitions":
+  {
+    "oauth2_auth":
+    {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://cloud.uipath.com/identity_/connect/authorize",
+      "tokenUrl": "https://cloud.uipath.com/identity_/connect/token",
+      "scopes":
+      {
+        "OR.Execution.Read OR.Folders.Read OR.Jobs.Write OR.Queues offline_access": "OR.Execution.Read OR.Folders.Read OR.Jobs.Write OR.Queues offline_access"
+      }
+    }
+  },
+  "security":
+  [
+    {
+      "oauth2_auth":
+      [
+        "OR.Execution.Read OR.Folders.Read OR.Jobs.Write OR.Queues offline_access"
+      ]
+    }
+  ],
+  "parameters":
+  {
+    "filter":
+    {
+      "in": "query",
```

Comment:
Re-use the oderby parameter you defined. Do this for all instances of filter, orderby and top.